zstd_vendor/CMakeLists.txt

```diff
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.5)
+project(zstd_vendor)
+
+list(INSERT CMAKE_MODULE_PATH 0 "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
+find_package(zstd)
+
+if (NOT zstd_FOUND)
+  set(extra_cmake_args)
+
+  if(DEFINED CMAKE_TOOLCHAIN_FILE)
+    list(APPEND extra_cmake_args "-DCMAKE_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE}")
+  endif()
+
+  include(ExternalProject)
+  set(EP_PREFIX zstd-1.4.4)
+  ExternalProject_Add(zstd-1.4.4
+    GIT_REPOSITORY https://github.com/facebook/zstd.git
+    GIT_TAG v1.4.4
+    CONFIGURE_COMMAND ${CMAKE_COMMAND} ${CMAKE_CURRENT_BINARY_DIR}/zstd/src/${EP_PREFIX}/build/cmake
```

Comment:
do you need this CONFIGURE_COMMAND?

Reply:
Yes, because zstd doesn't have the cmakelists.txt in the root directory. It's in [build/cmake](https://github.com/facebook/zstd/blob/dev/build/cmake/CMakeLists.txt)